airflow/utils/timezone.py

```diff
@@ -102,16 +102,18 @@ def convert_to_utc(value):
 
 
 @overload
-def make_aware(v: None, timezone: Optional["Timezone"] = None) -> None:
+def make_aware(value: None, timezone: Optional[Union["Timezone", dt.tzinfo]] = None) -> None:
```

Comment:
```suggestion
def make_aware(value: None, timezone: Optional[dt.tzinfo] = None) -> None:
```

Because `pendulum.tz.Timezone` is a subclass of `datetime.tzinfo`.